vtex-apps/store-header
# Upgrade packages with security vulnerabilities

#### What is the purpose of this pull request?

Upgrade packages with security vulnerabilities.

#### What problem is this solving?

Security vulnerabilities.

#### How should this be manually tested?
Run `yarn audit` in both the root and the `react` folder, and make sure there are no vulnerabilities found.

#### Screenshots or example usage
N/A.

#### Types of changes
- Security fix.
